.github/workflows/loki-update.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.METABASE_AUTOMATION_USER_TOKEN }}
       - name: Remove 'loki-update' label
-        uses: actions/github-script@v3
```

Comment:
This particular version is really old.
Is there a technical reason or a limitation that prevents us from using the latest version? @alxnddr